baseplate.go

```diff
@@ -246,30 +246,20 @@ func Serve(ctx context.Context, args ServeArgs) error {
 //       FancyName string `yaml:"fancy_name"`
 //     }
 //     var cfg myServiceCfg
-//     err := baseplate.ParseConfigYAML(reader, &cfg)
-//     // TODO: handle err
+//     if err := baseplate.ParseConfigYAML(&cfg); err != nil {
+//       log.Fatalf("Parsing config: %s", err)
+//     }
 //     ctx, bp, err := baseplate.New(baseplate.NewArgs{
-//       EdgeContextFactory: ...,
```

Comment:
this is still required so we probably should keep this line?

Reply:
added back, though I'm also throwing in the `edgecontext.Factory(...)`